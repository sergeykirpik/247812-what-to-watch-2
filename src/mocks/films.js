const films = [
  {
    id: 1,
    title: `Fantastic Beasts: The Crimes of Grindelwald`,
    previewImage: `img/fantastic-beasts-the-crimes-of-grindelwald.jpg`,
  },
  {
    id: 2,
    title: `Bohemian Rhapsody`,
    previewImage: `img/bohemian-rhapsody.jpg`,
  },
  {
    id: 3,
    title: `Macbeth`,
    previewImage: `img/macbeth.jpg`,
  },
  {
    id: 4,
    title: `Aviator`,
    previewImage: `img/aviator.jpg`,
  },
  {
    id: 5,
    title: `We need to talk about Kevin`,
    previewImage: `img/we-need-to-talk-about-kevin.jpg`,
  },
  {
    id: 6,
    title: `What We Do in the Shadows`,
    previewImage: `img/what-we-do-in-the-shadows.jpg`,
  },
  {
    id: 7,
    title: `Revenant`,
    previewImage: `img/revenant.jpg`,
  },
  {
    id: 8,
    title: `Johnny English`,
    previewImage: `img/johnny-english.jpg`,
  },
  {
    id: 9,
    title: `Shutter Island`,
    previewImage: `img/shutter-island.jpg`,
  },
  {
    id: 10,
    title: `Pulp Fiction`,
    previewImage: `img/pulp-fiction.jpg`,
  },
  {
    id: 11,
    title: `No Country for Old Men`,
    previewImage: `img/no-country-for-old-men.jpg`,
  },
  {
    id: 12,
    title: `Snatch`,
    previewImage: `img/snatch.jpg`,
  },
  {
    id: 13,
    title: `Moonrise Kingdom`,
    previewImage: `img/moonrise-kingdom.jpg`,
  },
  {
    id: 14,
    title: `Seven Years in Tibet`,
    previewImage: `img/seven-years-in-tibet.jpg`,
  },
  {
    id: 15,
    title: `Midnight Special`,
    previewImage: `img/midnight-special.jpg`,
  },
  {
    id: 16,
    title: `War of the Worlds`,
    previewImage: `img/war-of-the-worlds.jpg`,
  },
  {
    id: 17,
    title: `Dardjeeling Limited`,
    previewImage: `img/dardjeeling-limited.jpg`,
  },
  {
    id: 18,
    title: `Orlando`,
    previewImage: `img/orlando.jpg`,
  },
  {
    id: 19,
    title: `Mindhunter`,
    previewImage: `img/mindhunter.jpg`,
  },
  {
    id: 20,
    title: `Midnight Special`,
    previewImage: `img/midnight-special.jpg`,
  }
];

export default films;
