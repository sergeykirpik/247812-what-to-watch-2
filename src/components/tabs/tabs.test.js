import React from 'react';
import renderer from 'react-test-renderer';
import Tabs from './tabs.jsx';

import films from '../../mocks/films2';

it(`renders correctly`, () => {
  const tree = renderer.create(
      <Tabs film={films[0]} />
  ).toJSON();
  expect(tree).toMatchSnapshot();
});
