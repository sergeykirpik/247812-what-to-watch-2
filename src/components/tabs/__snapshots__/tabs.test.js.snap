// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
Array [
  <nav
    className="movie-nav movie-card__nav"
  >
    <ul
      className="movie-nav__list"
    >
      <li
        className="movie-nav__item"
      >
        <a
          className="movie-nav__link"
          href="#"
          onClick={[Function]}
        >
          Overview
        </a>
      </li>
      <li
        className="movie-nav__item movie-nav__item--active"
      >
        <a
          className="movie-nav__link"
          href="#"
          onClick={[Function]}
        >
          Details
        </a>
      </li>
      <li
        className="movie-nav__item"
      >
        <a
          className="movie-nav__link"
          href="#"
          onClick={[Function]}
        >
          Reviews
        </a>
      </li>
    </ul>
  </nav>,
  <div
    className="movie-card__text movie-card__row"
  >
    <div
      className="movie-card__text-col"
    >
      <p
        className="movie-card__details-item"
      >
        <strong
          className="movie-card__details-name"
        >
          Director
        </strong>
        <span
          className="movie-card__details-value"
        >
          Sergio Leone
        </span>
      </p>
      <p
        className="movie-card__details-item"
      >
        <strong
          className="movie-card__details-name"
        >
          Starring
        </strong>
        <span
          className="movie-card__details-value"
        >
          Robert De Niro,
          <br />
          James Woods,
          <br />
          Elizabeth McGovern
        </span>
      </p>
    </div>
    <div
      className="movie-card__text-col"
    >
      <p
        className="movie-card__details-item"
      >
        <strong
          className="movie-card__details-name"
        >
          Run Time
        </strong>
        <span
          className="movie-card__details-value"
        >
          3h 49m
        </span>
      </p>
      <p
        className="movie-card__details-item"
      >
        <strong
          className="movie-card__details-name"
        >
          Genre
        </strong>
        <span
          className="movie-card__details-value"
        >
          Crime
        </span>
      </p>
      <p
        className="movie-card__details-item"
      >
        <strong
          className="movie-card__details-name"
        >
          Released
        </strong>
        <span
          className="movie-card__details-value"
        >
          1984
        </span>
      </p>
    </div>
  </div>,
]
`;
